test/metabase/events/revision_test.clj

```diff
@@ -267,44 +252,41 @@
                                            :dashboard_id dashboard-id}])
               :is_reversion false
               :is_creation  false}
-             (mt/derecordize
-              (t2/select-one [Revision :model :model_id :user_id :object :is_reversion :is_creation]
-                :model    "Dashboard"
-                :model_id dashboard-id)))))))
+             (t2/select-one [Revision :model :model_id :user_id :object :is_reversion :is_creation]
+                            :model    "Dashboard"
+                            :model_id dashboard-id))))))
 
 (deftest dashboard-delete-tabs-test
-  (testing ":dashboard-delete-tabs"
+  (testing :event/dashboard-remove-tabs
     (t2.with-temp/with-temp
       [:model/Dashboard     {dashboard-id :id, :as dashboard} {:name "A dashboard"}
        :model/DashboardTab  {dashtab-id :id}                  {:name         "First tab"
                                                                :position     0
                                                                :dashboard_id dashboard-id}]
       (t2/delete! :model/DashboardTab dashtab-id)
-      (revision/process-revision-event! {:topic :dashboard-delete-tabs
```

Comment:
Here's another typo the new code caught. The event was always `:dashboard-remove-tabs`. `:dashboard-delete-tabs` was never a valid event.